src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -52,11 +52,19 @@
 helps['appservice plan show'] = """
 type: command
 short-summary: Get the app service plans for a resource group or a set of resource groups.
+examples:
+  - name: Get the app service plans for a resource group or a set of resource groups. (commonly used with --output)
+    text: 'az appservice plan show --name MyAppServicePlan --resource-group MyResourceGroup  '
+    crafted: true
 """
 
 helps['appservice plan update'] = """
 type: command
 short-summary: Update an app service plan. See https://docs.microsoft.com/en-us/azure/app-service/app-service-plan-manage#move-an-app-to-another-app-service-plan to learn more
+examples:
+  - name: Update an app service plan. (crafted)
```

Comment:
(crafted) is unnecessary here. The customer won't have any idea what that means.